### PR TITLE
Docker file to create an RPI image

### PR DESCRIPTION
**Pull request by romerod**
A docker file to create an image which can be directly run on an RPI.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 target
 cache
 protocol/target
+contrib
```
[the remark this answers is not in the document]
Comment (awiouy) [reply .dockerignore]: Then how do you pull the dockerfiles from github?

[the remark this answers is not in the document]
Comment (romerod) [reply .dockerignore]: I don't understand the question, it's the docker ignore file not the git one, could you explain?

[the remark this answers is not in the document]
Comment (awiouy) [reply .dockerignore]: Meh... my bad

**Comment by sashahilton00** [comment]
LGTM, will merge if no objections.

File: .dockerignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (bricedp) [reply .dockerignore]: The contrib folder is used in the image for the Raspberry Pi build — see #243.

**Pull request merged**